bin/heatmapper

```diff
@@ -119,15 +119,26 @@ def prepare_layout(hm_matrix, heatmapsize, showSummaryPlot, showColorbar):
 
     numcols = hm_matrix.get_num_samples()
     numrows = hm_matrix.get_num_groups()
+    if perGroup:
+        temp = numcols
```

Comment:
I switch between enough languages that this is the only way I can maintain sanity :p